build.sbt

```diff
@@ -4,9 +4,9 @@ organization := "com.microsoft.sqlserver.jdbc.spark"
 
 version := "1.0.0"
 
-scalaVersion := "2.11.12"
-
-val sparkVersion = "2.4.6"
+scalaVersion := "2.12.11"
+ThisBuild / useCoursier := false
+val sparkVersion = "3.0.0"
```

Comment:
Would need to support sparkVersion 2.4/Scala 2.11 combo as well.

Reply:
Ok, I will look into supporting both scenarios.

Reply:
I think you can use something like
`crossScalaVersions := Seq("2.12.10", "2.11.12")`
to do this

Reply:
@rajmera3 it won't work, because there is no Spark 3.0 with Scala 2.11.  Here we need to have a combo of (Spark 2.4 + Scala 2.11) & (Spark 3.0 + Scala 2.12)

Reply:
One option here is to make main line as Spark3.0/Scala.2.12 and stable/old version to a separate branch e.g. Spark2.4 branch.